README.md

```diff
@@ -1,8 +1,11 @@
-# GovukTemplate
+# GOV.UK Template
 
 This provides a template containing the GOV.UK header and footer, and associated assets.
```

Comment:
This might be outside of the scope of this PR, but it would be good for the main readme to contain something about *who* template is for. It's not clear (to me) that this is outward facing for services rather than just for GDS use.

 eg 'GOV.UK Template is meant for all services running under the service.gov.uk domain....'